### PR TITLE
[CPU] Upgrade CPU func tests to 2.0

### PR DESCRIPTION
**Pull request by yangwang201911**
### Details:
 - updated test case "CompareWithRefs".
 - updated test case "PSROIPoolingLayerCPUTest". 

### Tickets:
 - CVS-125924

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/convert_to_plugin_specific_node.cpp [hunk 1]
```diff
@@ -45,74 +46,75 @@ class ConvertToPluginSpecificNode : public testing::WithParamInterface<ConvertTo
...
+        utils::EltwiseTypes nodeType;
         size_t port;
 
         std::tie(nonConstShape, constShape, prc, nodeType, port, constNodeNum) = this->GetParam();
         OPENVINO_ASSERT(shape_size(constShape) == 1);
 
-        const auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(prc);
-        const auto param = std::make_shared<ngraph::opset1::Parameter>(ngPrc, ngraph::Shape(nonConstShape));
-        const auto constNode = builder::makeConstant(ngPrc, ngraph::Shape(constShape), std::vector<float>{}, true);
+        const auto ngPrc = prc;
```
Comment (zhaixuejun1993) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/convert_to_plugin_specific_node.cpp:57]: can use "prc" directly, remove "ngPrc"

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp [hunk 2]
```diff
@@ -64,120 +61,121 @@ class PSROIPoolingLayerCPUTest : public testing::WithParamInterface<PSROIPooling
...
+                                                                mode);
         psroi->get_rt_info() = getCPUInfo();
-        selectedType = getPrimitiveType() + "_" + inPrc.name();
+        std::ostringstream result;
+        result << inType;
+        selectedType = getPrimitiveType() + "_" + result.str();
 
-        threshold = 1e-2f;
+        rel_threshold = 1e-2f;
         const ngraph::ResultVector results{std::make_shared<ngraph::opset3::Result>(psroi)};
```
Comment (riverlijunjie) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp:104]:
```suggestion
        const ov::ResultVector results{std::make_shared<ov::op::v0::Result>(psroi)};
```

Comment (yangwang201911) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp:104]: updated

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/convert_to_plugin_specific_node.cpp [hunk 3]
```diff
@@ -8,6 +8,7 @@
 #include "test_utils/cpu_test_utils.hpp"
 
 using namespace CPUTestUtils;
+using namespace ov::test::utils;
```
[the remark this answers is not in the document]
Comment (yangwang201911) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/convert_to_plugin_specific_node.cpp:11]: Updated.

**Review by zhaixuejun1993 — APPROVED** [review]

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp [hunk 4]
```diff
@@ -64,120 +61,121 @@ class PSROIPoolingLayerCPUTest : public testing::WithParamInterface<PSROIPooling
...
-                 spatialScale, spatialBinsX, spatialBinsY, mode) = psroiPoolingParams;
+        inType = outType = netPrecision;
+        std::tie(featureMapShape, proposal, outputDim, groupSize, spatialScale, spatialBinsX, spatialBinsY, mode) =
+            psroiPoolingParams;
 
-
-        ngraph::Shape proposalShape = { proposal.size() / 5, 5 };
+        ov::Shape proposalShape = {proposal.size() / 5, 5};
 
         auto coords = ngraph::builder::makeConstant<float>(ngraph::element::f32, proposalShape, proposal);
```
Comment (riverlijunjie) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp:86]:
```suggestion
        auto coords = ngraph::builder::makeConstant<float>(ov::element::f32, proposalShape, proposal);
```

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp [hunk 5]
```diff
@@ -64,120 +61,121 @@ class PSROIPoolingLayerCPUTest : public testing::WithParamInterface<PSROIPooling
...
+
+public:
+    std::vector<float> proposal;
+    ov::Shape featureMapShape;
+    size_t spatialBinsX;
+    size_t spatialBinsY;
+    float spatialScale;
+    size_t groupSize;
+    size_t outputDim;
+    std::string mode;
```
Comment (vurusovs) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp:116]: Let's make it local for `SetUp()`

**Review by vurusovs — CHANGES_REQUESTED** [review]

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp (hunk 5 above)
Comment (yangwang201911) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/psroi_pooling.cpp]: sure. Updated.

**Review by vurusovs — APPROVED** [review]

**Review by riverlijunjie — APPROVED** [review]

**Comment by zhaixuejun1993** [comment]
replaced by https://github.com/openvinotoolkit/openvino/pull/21866
close

**Pull request closed without merging**